packages/vats/src/core/types-ambient.d.ts

```diff
@@ -507,3 +518,22 @@ type HttpVat = ERef<
 type UploadsVat = ERef<
   ReturnType<typeof import('@agoric/solo/src/vat-uploads.js').buildRootObject>
 >;
+
+type WellKnownVats = SwingsetVats & {
+  bank: BankVat;
+  board: BoardVat;
+  bridge: ChainStorageVat;
+  localchain: LocalChainVat;
+  ibc: IBCVat;
+  mints: MintsVat;
+  network: NetworkVat;
+  orchestration: ERef<
+    ReturnType<
+      typeof import('@agoric/orchestration/src/vat-orchestration.js').buildRootObject
+    >
+  >;
```

Comment:
vat-orchestration exports this
```suggestion
  orchestration: OrchestrationVat;
```

Reply:
Okay, adapted.